experimenter/experimenter/segments/__init__.py

```diff
@@ -7,6 +9,22 @@
 from experimenter.experiments.constants import NimbusConstants
 
 
+@dataclass
+class MockCommit:
+    committed_date: int
```

Comment:
Looking at metric_config_parser, this doesnt look like it has enough information:


```python
        for outcome_file in files_path.glob(f"{OUTCOMES_DIR}/*/*.toml"):
            commit_hash = next(repo.iter_commits("HEAD", paths=outcome_file)).hexsha
```

[source](
https://github.com/mozilla/metric-hub/blob/7321591a45793f61f5a63d301eb24688374dfb9a/lib/metric-config-parser/metric_config_parser/config.py#L424)

Reply:
`commit_hash` isn't required for `Outcome`, but I've come around to the "just do this in metric-config-parser" now, so no longer important.